pkgs/top-level/all-packages.nix

```diff
@@ -15619,9 +15619,13 @@ in
   bluez = bluez5;
 
   bluezFull = bluez.override {
-    enableWiimote = true;
+    enableHealth = true;
+    enableMesh = true;
     enableMidi = true;
+    enableNfc = true;
+    enableSap = true;
     enableSixaxis = true;
+    enableWiimote = true;
```

Comment:
why don't you just default all the values to true? and get rid of passing the arguments explicitly

Reply:
I would be in favor of just aliasing bluezFull to bluez, and just having 1 derivation